### PR TITLE
Fix nightly workflow is almost always red

### LINKED ISSUE #3123: Nightly workflow is almost always red
Nightly workflow is always red on android 21, 22, 25, 28.

### PR DESCRIPTION
**Pull request by MohitMaliFtechiz**
Fixes #3123

### CODE REVIEW

**Comment by MohitMaliFtechiz** [comment]
hi @gouri-panda , this PR is now ready for review. please review this PR.

**Comment by gouri-panda** [comment]
Ok @MohitMaliFtechiz

File: app/src/androidTest/java/org/kiwix/kiwixmobile/main/TopLevelDestinationTest.kt [hunk 1]
```diff
@@ -46,11 +53,7 @@ class TopLevelDestinationTest : BaseActivityTest() {
         clickFileTransferIcon {
         }
       }
-      clickDownloadOnBottomNav {
-        assertLibraryListDisplayed()
-        clickOnGlobeIcon {
-        }
-      }
+      clickDownloadOnBottomNav(OnlineLibraryRobot::assertLibraryListDisplayed)
```
Comment (gouri-panda) [inline app/src/androidTest/java/org/kiwix/kiwixmobile/main/TopLevelDestinationTest.kt:56]: We shouldn't delete the locale language test cases.

File: app/src/androidTest/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryRobot.kt [hunk 2]
```diff
@@ -34,8 +32,7 @@ class OnlineLibraryRobot : BaseRobot() {
     isVisible(ViewId(R.id.libraryList))
   }
 
-  fun clickOnGlobeIcon(func: LanguageRobot.() -> Unit) {
+  fun clickOnGlobeIcon() {
     clickOn(ViewId(R.id.select_language))
-    language(func)
```
Comment (gouri-panda) [inline app/src/androidTest/java/org/kiwix/kiwixmobile/nav/destination/library/OnlineLibraryRobot.kt:39]: We should find the bug instead of deleting tests.

**Review by gouri-panda — CHANGES_REQUESTED** [review]

File: app/src/androidTest/java/org/kiwix/kiwixmobile/main/TopLevelDestinationTest.kt (hunk 1 above)
Comment (MohitMaliFtechiz) [reply app/src/androidTest/java/org/kiwix/kiwixmobile/main/TopLevelDestinationTest.kt:56]: hi @gouri-panda ,

we are not deleting this test we are running this test separately https://github.com/kiwix/kiwix-android/blob/develop/app/src/androidTest/java/org/kiwix/kiwixmobile/language/LanguageFragmentTest.kt here.

**Comment by kelson42** [comment]
@gouri-panda Finally good to you?

**Review by gouri-panda — APPROVED** [review]
@MohitMaliDeveloper Thanks!

**Pull request merged**